### PR TITLE
docs: add 1.11.0 changes

### PR DESCRIPTION
**Pull request by GGabriele**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +36,41 @@
...
+
+> Release date: 2022/02/17
+
+### Added
+
+- decK now populates core entities defaults from their schema leveraging the Admin API.
+  [#573](https://github.com/Kong/deck/pull/573)
+- decK now populates plugins defaults from their schema leveraging the Admin API.
+  [#562](https://github.com/Kong/deck/pull/562)
+- decK prevents users to interact with any internal-only Konnect plugins.
```
Comment (rainest) [inline CHANGELOG.md:49]:
```suggestion
- decK prevents user interaction with any internal-only Konnect plugins.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -35,6 +36,41 @@
...
+  `--tls-client-cert`/`--tls-client-cert-file` and
+  `--tls-client-key`/`--tls-client-key-file` flags.
+  [#509](https://github.com/Kong/deck/pull/509)
+- decK `validate` now has a `--online` flag enabling entities validation against
+  the Admin API. This lets users validate their configuration before deploying it.
+  [#502](https://github.com/Kong/deck/pull/502)
+
+### Fixes
+
+- decK now can set zero values in entity fields.
```
Comment (rainest) [inline CHANGELOG.md:61]:
```suggestion
- decK now can set zero values (`false`, `""`, `0`) in entity fields.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -35,6 +36,41 @@
...
+  [#580](https://github.com/Kong/deck/pull/580)
+- fix unexpected behaviour when running `sync` against multiple workspaces at once.
```
Comment (rainest) [inline CHANGELOG.md:63]:
```suggestion
- Attempting to run `sync` on multiple workspaces now returns an error rather th
  an applying a single workspace to all entities.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -35,6 +36,41 @@
...
+  [#576](https://github.com/Kong/deck/pull/576)
+- skip `consumers` when using `--skip-consumers` with the `sync` command.
+  [#559](https://github.com/Kong/deck/pull/559)
+- analytics flag now works with the `konnect ping` command.
+  [#569](https://github.com/Kong/deck/pull/569)
+- fix unexpected error with `--select-tag` and multiple files.
```
Comment (rainest) [inline CHANGELOG.md:69]:
```suggestion
- Duplicate `select_tags` in separate files and `--select-tags` no longer cause errors.
```

**Review by rainest — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -35,6 +36,41 @@
...
+  [#573](https://github.com/Kong/deck/pull/573)
+- decK now populates plugins defaults from their schema leveraging the Admin API.
+  [#562](https://github.com/Kong/deck/pull/562)
+- decK prevents users to interact with any internal-only Konnect plugins.
+  [#564](https://github.com/Kong/deck/pull/564)
+- decK now supports mTLS Kong Admin API authentication via
+  `--tls-client-cert`/`--tls-client-cert-file` and
+  `--tls-client-key`/`--tls-client-key-file` flags.
+  [#509](https://github.com/Kong/deck/pull/509)
+- decK `validate` now has a `--online` flag enabling entities validation against
```
Comment (lena-larionova) [inline CHANGELOG.md:55]:
```suggestion
- decK `validate` now has an `--online` flag enabling entity validation against
```

File: CHANGELOG.md (hunk 3 above)
Comment (lena-larionova) [reply CHANGELOG.md:63]: ^ typo/accidental line break mid-word, otherwise the suggestion looks good.

File: CHANGELOG.md [hunk 6]
```diff
@@ -35,6 +36,41 @@
...
+  the Admin API. This lets users validate their configuration before deploying it.
+  [#502](https://github.com/Kong/deck/pull/502)
+
+### Fixes
+
+- decK now can set zero values in entity fields.
+  [#580](https://github.com/Kong/deck/pull/580)
+- fix unexpected behaviour when running `sync` against multiple workspaces at once.
+  [#576](https://github.com/Kong/deck/pull/576)
+- skip `consumers` when using `--skip-consumers` with the `sync` command.
```
Comment (lena-larionova) [inline CHANGELOG.md:65]:
```suggestion
- Skip `consumers` when using `--skip-consumers` with the `sync` command.
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -35,6 +36,41 @@
...
+  [#559](https://github.com/Kong/deck/pull/559)
+- analytics flag now works with the `konnect ping` command.
```
Comment (lena-larionova) [inline CHANGELOG.md:67]:
```suggestion
- The `--analytics` flag now works with the `konnect ping` command.
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -35,6 +36,41 @@
...
+  [#569](https://github.com/Kong/deck/pull/569)
+- fix unexpected error with `--select-tag` and multiple files.
+  [#571](https://github.com/Kong/deck/pull/571)
+- enable `--header` flag for Konnect commands
```
Comment (lena-larionova) [inline CHANGELOG.md:71]:
```suggestion
- The `--header` flag is now enabled for Konnect commands.
```

**Comment by GGabriele** [comment]
@lena-larionova @rainest ready for another pass

**Review by lena-larionova — APPROVED** [review]

**Comment by hbagdi** [comment]
@GGabriele Please convert a draft to a mergable PR as it gets "closer" to polished.

Please merge on release day and follow these steps for release: https://github.com/Kong/deck/blob/main/docs/development/release.md
Travis, it probably makes sense for you to do the release and have @GGabriele shadow on a call.

**Review by rainest — APPROVED** [review]

**Pull request merged**